### ISSUE TITLE
🏁 Wake Me Up ! ↔ v23.4.2

### ISSUE THREAD

**Issue by ElTipejoLoco** [itchio/itch-compatibility-watchlist#1190]
:rotating_light: game [Wake Me Up !](https://jpe.itch.io/wake-me-up) is broken for me.

:book: Here's a truncated [debug log](https://gist.github.com/ElTipejoLoco/de558db82990966a5a17dfd625e73137).

:running: Hey, it looks like the itch app's running into some sort of problem when it comes to creating or reading a folder for the game Wake Me Up !, or at least that's what I gleaned from what I checked of the itch.txt installation log. I truncated it heavily to spare you searching through the whole thing, but if you need the original log file(s) anyway, let me know.

I can confirm that a "Wake Me Up" folder was successfully created by the installation attempt, but it completely lacks the ".itch" hidden system folder and its intended contents that the desktop app sort of expects to exist there, so I'm guessing something must be up with that.

**Comment by fasterthanlime** [comment]
Does the "Wake Me Up" folder on disk have a space at the end or not?

ie. is it "Wake Me Up" or "Wake Me Up " ?

**Comment by ElTipejoLoco** [comment]
The folder the desktop app created lacks the space.

**Comment by ElTipejoLoco** [comment]
[Halloween Cat ! !](https://pistachioegg.itch.io/halloween-cat) is suffering from the same problem, it looks like. Do you want a debug log for it as well, or is a fix for this issue already in the works in the upcoming version 25?

**Comment by fasterthanlime** [comment]
> is a fix for this issue already in the works in the upcoming version 25?

Yes, v25 doesn't have this problem because it uses the game's "slug" (the `halloween-cat` in https://pistachioegg.itch.io/halloween-cat) rather than the game's title (which may contain multiple non-alphanumeric characters - or even no alphanumeric characters at all).

Sorry for the inconvenience!
